dataland-backend/src/main/kotlin/org/dataland/datalandbackend/controller/CompanyDataController.kt

```diff
@@ -211,4 +211,10 @@ class CompanyDataController(
         logger.info("Received a request to check if user with Id $userId is data owner of company with Id $companyId.")
         dataOwnersManager.checkUserCompanyCombinationForDataOwnership(companyId.toString(), userId.toString())
     }
+
+    override fun postDataOwnershipRequest(companyId: UUID, comment: String?) {
```

Comment:
Hmm. The correlation ID would be generated here. But we would not forward it to any other service. We could put it in the email. What do you think? I have no strong opinion.

Reply:
So we could only log it in the method that sends the email? Question is would that help us when we would have to debugg it?

Reply:
To me it only makes sense if we add the correlation id to the email. Because all other logs we can still associate with another by the thread / process id which is printed after "INFO"/"WARN"/... in the log messages.

Reply:
ok, then lets leave it out, because i don't think it belongs in the email itself